src/app/presentation/arrows/arrows.component.ts

```diff
@@ -16,6 +16,13 @@ export class ArrowsComponent implements OnInit {
     this.presentation.nextSlide();
   }
 
+  canGoNext(): boolean {
+    return this.presentation.activeSlideIndex + 1 < this.presentation.totalSlides;
+  }
```

Comment:
Would be cool move those to the presentation and reuse for next/previous slide?
https://github.com/kirjs/angular-presentation/blob/master/src/app/presentation/presentation/presentation.component.ts#L51